tests/js/spec/views/events/searchBar.spec.jsx

```diff
@@ -78,7 +78,7 @@ describe('SearchBar', function() {
 
     expect(tagValuesMock).toHaveBeenCalledWith(
       '/organizations/org-slug/tags/gpu/values/',
-      expect.objectContaining({query: {project: [1, 2], statsPeriod: '14d'}})
+      expect.objectContaining({query: {project: ['1', '2'], statsPeriod: '14d'}})
```

Comment:
Wonder if this will cause issues elsewhere

Reply:
Hmm, I thought to convert it and change the tests because I had assumed these are getting converted to strings regardless when constructing the query or fetchingTagValues.